src/main/java/roomescape/repository/ReservationTimeJdbcRepository.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.repository;
+
+import java.time.LocalTime;
+import java.util.List;
+import javax.sql.DataSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.ReservationTime;
+
+@Repository
+public class ReservationTimeJdbcRepository implements ReservationTimeRepository{
+
+    private static final String TABLE_NAME = "reservation_time";
+
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert simpleJdbcInsert;
+
+    private final RowMapper<ReservationTime> reservationTimeRowMapper = (resultSet, rowNum) -> {
+        ReservationTime reservationTime
+                = new ReservationTime(
+                resultSet.getLong("id"),
+                LocalTime.parse(resultSet.getString("start_at"))
+        );
+        return reservationTime;
+    };
```

Comment:
```suggestion
    private final RowMapper<ReservationTime> reservationTimeRowMapper = (resultSet, rowNum) -> new ReservationTime(
            resultSet.getLong("id"),
            LocalTime.parse(resultSet.getString("start_at"))
    );
```
람다 표현식으로 작성해볼 수 있겠네요 👀